code/game/objects/items/storage/boxes.dm

```diff
@@ -74,6 +74,45 @@
 		return 0
 	return ..()
 
+//Locker overloading issue solving boxes
+/obj/item/storage/box/suitbox
+	name = "compression box of invisible outfits"
+	desc = "a box with bluespace compression technology that nanotrasen has approved, but this is extremely heavy... If you're glued with this box, pull out of the contents and fold the box."
+	w_class = WEIGHT_CLASS_HUGE
+	drag_slowdown = 4 // do not steal by dragging
+	/* Note for the compression box:
+		Do not put any box (or suit) into this box, or it will allow infinite storage.
+		non-storage items are only legit for this box. (suits are storage too, so, no.)
+		nor it will allow a glitch when you can access different boxes at the same time.
+		examples exist in `closets/secure/security.dm` */
+
+/obj/item/storage/box/suitbox/pickup(mob/user)
+	. = ..()
+	user.add_movespeed_modifier(MOVESPEED_ID_SLOW_SUITBOX, update=TRUE, priority=100, multiplicative_slowdown=4)
+
+/obj/item/storage/box/suitbox/dropped(mob/living/user)
+	..()
+	var/box_exists = FALSE
+	for(var/obj/item/storage/box/suitbox/B in user.get_contents())
+		box_exists = TRUE // `var/obj/item/storage/box/suitbox/B` is already type check
```

Comment:
this can be simplified. move the remove_movespeed_modifier up here, then replace `break` with `return`.
Problem solved.

Reply:
nevermind, it's supposed to be if no boxes. silly me.